Gemfile

```diff
@@ -5,6 +5,8 @@ gemspec name: 'metasploit-framework'
 
 gem 'sqlite3', '~>1.3.0'
 
+gem 'memory_profiler'
```

Comment:
Can move this into `group :development, :test do`?

Reply:
Sure